Makefile

```diff
@@ -5,8 +5,8 @@ DB_NAME := develop_db
 database: #See what's in database
 	docker exec -it postgres_db bash -c "psql -U $(DB_USERNAME) -d $(DB_NAME)"
 
-web: #See what's in database
+web: #See what's in django
 	docker exec -it django_container bash
 
-nginx: #See what's in database
+ws: #See what's in webserver
```

Comment:
rename nginx to ws(Web Server for short) so make command does confuse it with docker